gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -414,6 +414,10 @@
   public static final HadoopConfigurationProperty<RedactedString> GCS_ENCRYPTION_KEY_HASH =
       new HadoopConfigurationProperty<>("fs.gs.encryption.key.hash");
 
+  /** Configuration key to enable publishing metrics to Google cloud monitoring. */
+  public static final HadoopConfigurationProperty<Boolean> GCS_METRICS_ENABLE =
+      new HadoopConfigurationProperty<>("fs.gs.metrics.enable", false);
```

Comment:
Probably better to make it more specific if this is gRPC or Cloud Monitoring specific, even better is to make this an enum, so we can support multiple metrics sinks in the future, if necessary

Reply:
I do not think that we need to have 2 flags, 1st to enable metrics and 2nd to chose sinks - sinks configuration is sufficient, I think. So we can already change this to `fs.gs.metrics.sink=NONE|CLOUD_MONITORING`. Not sure if we actually want to support multiple sinks at the same time, if no, we can live this as a single-value enum property

Reply:
I meant support multiple sinks (not at the same time). We could publish to Cloud-monitoring/ Prometheus/ ..

Ack the changes with single-value enum property